tsconfig.json

```
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+      "target": "es6",
+      "module": "commonjs",
+      "rootDir": ".",
+      "resolveJsonModule": true,
+      "allowJs": true,
+      "checkJs": false,
+      "outDir": "dist",
+      "esModuleInterop": true,
+      "forceConsistentCasingInFileNames": true,
+      "strict": true,
+      "declaration": true,
+      "noImplicitAny": false,
+      "noEmitOnError": false,
+      "sourceMap": true,
+      "skipLibCheck": true
```

Need to add the es2021 lib to use the target
```suggestion
      "skipLibCheck": true,
      "lib": ["es2021"]
```